app/controllers/reviewables_controller.rb

```diff
@@ -69,6 +74,8 @@ def index
           total_rows_reviewables: total_rows,
           types: meta_types,
           reviewable_types: Reviewable.types,
+          score_types:
+            ReviewableScore.types.map { |k, v| { id: v, name: ReviewableScore.type_title(k) } },
```

Comment:
Oh maybe this is the place we need to exclude `notify_user`

Reply:
yeah will do so... I think this is the simplest way of resolving this